src/pages/billing/CreateGCPBillingProject.ts

```diff
@@ -11,8 +11,6 @@ import { billingProjectNameValidator } from 'src/pages/billing/billing-utils';
 import { GoogleBillingAccount } from 'src/pages/billing/models/GoogleBillingAccount';
 import validate from 'validate.js';
 
-const BillingAccountSelect = Select as typeof Select<GoogleBillingAccount>;
```

Comment:
@nawatts I was not able to figure out how to add appropriate type definitions to `VirtualizedSelect` in order to keep this type of definition.